tools/deployment/make_linux_package.sh

```diff
@@ -51,7 +52,6 @@ DEBUG_PREFIX=$WORKING_DIR/debug
 
 function usage() {
   fatal "Usage: $0 -t deb|rpm -i REVISION -d DEPENDENCY_LIST
-
```

Comment:
This is as confusing as it is infuriating ;) I've never had these issues with any other files or commits before, and I've used Sublime for years. Just examined the file in both vim and Atom, there is no trailing space showing. Output from cat -A is below: